rpcs3/Emu/RSX/GL/GLGSRender.h

```diff
@@ -18,9 +18,11 @@ struct work_item
 	std::mutex guard_mutex;
 	
 	u32  address_to_flush = 0;
-	bool processed = false;
-	bool result = false;
-	bool received = false;
+	gl::texture_cache::cached_rtt_section *section_to_flush = nullptr;
+
+	volatile bool processed = false;
```

Comment:
They aren't atomic, since the thread that receives this is waiting on the condition variable. Its just to keep the compiler from doing any shady optimizations.